nilearn/conftest.py

```diff
@@ -243,6 +244,14 @@ def img_3d_mni():
     return _img_3d_mni()
 
 
+@pytest.fixture()
+def img_3d_mni_as_file(tmp_path):
+    """Return path to a random 3D Nifti1Image in MNI space saved to disk."""
+    filename = tmp_path / "img.nii"
```

Comment:
Just curious: how does the `/` operation work on two `str` (I think the first one is a string?)

Reply:
tmp_path is a pathlib.Path object 

It is one of the fixture provided by pytest
https://docs.pytest.org/en/8.0.x/how-to/tmp_path.html

so here we are just in "classic" Path territory. 😉

Reply:
I see that's cool. Have been using `os` functions so far, but `Path` looks interesting. Thanks!